develop/reference_dml.rst

```diff
@@ -127,12 +145,6 @@ Once we create this new distributed table, we can then run :code:`INSERT INTO ..
     WHERE site_id = 5 AND
       day >= date '2016-01-01' AND day < date '2017-01-01';
 
-It's worth noting that for :code:`INSERT INTO ... SELECT` to work on distributed tables, Citus requires the source and destination table to be co-located. In summary:
```

Comment:
Why delete these?

Reply:
Maybe reword this to for "on-conflict clause" to work.. we require colocation etc.

Reply:
Moved the colocation required warning a little later where it reads better.